lobster/tests/cstylecheatsheettest.lobster

```diff
@@ -0,0 +1,81 @@
+import std
```

Comment:
I don't think we need this file in here, generating `c_style_cheat_sheet.md` would be enough of a test already?

Reply:
Since `c_style_cheat_sheet.md` isn't actually generated (yet?), I just included it so you or anyone else could see that the examples in the cheat sheet actually do work and were tested.